components/ListIntegrations.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Text, View, TouchableOpacity, Image } from "react-native";
+import ScrollView from "../assets/ScrollView.imageset/ScrollView150.png";
+import TableView from "../assets/TableView.imageset/TableView150.png";
+import CollectionView from "../assets/CollectionView.imageset/CollectionView150.png";
+import WebView from "../assets/ScrollView.imageset/ScrollView150.png";
+import styles from "../styles/styles";
+
+const ListIntegrations = ({ navigation }) => {
+  function navigateToDemo(navigation) {
+    return navigation.navigate("Demo");
+  }
+
+  const tabIntegrationTypes = [
+    "ScrollView",
+    "TableView",
+    "CollectionView",
+    "WebView",
+  ];
+
+  const tabIntegrationImages = [ScrollView, TableView, CollectionView, WebView];
```

Comment:
You better make an object here
```suggestion
const tabIntegrations = [
  { type: 'ScrollView', image: ScrollView },
  { type: 'TableView', image: TableView },
  { type: 'CollectionView', image: CollectionView },
  { type: 'WebView', image: WebView },
]
```